### PR TITLE
React Query - 3 : added query for get original url

### PR DESCRIPTION
**Pull request by sahsisunny**
## Issue Ticket Number

- #56

## Description

Added query for get original url from short url and removed unused code.

**Breaking Changes**

-   [ ] Yes
-   [x] No

**Development Tested?**

-   [x] Yes
-   [ ] No

**Tested in Staging?**

-   [ ] Yes
-   [x] No

**Under Feature Flag**

-   [ ] Yes
-   [x] No

**Database Changes**

-   [ ] Yes
-   [x] No

## Test Coverage

| File                         | Statements | Branches | Functions | Lines | Uncovered|
| ---------------------------- | ---------- | -------- | --------- | ------ |------|
| **src/components/Redirect**  | 100        | 100      | 100       | 100    ||
| LoaderTimer.tsx              | 100        | 100      | 100       | 100    ||
| NotFound.tsx                 | 100        | 100      | 100       | 100    ||
| RedirectFooter.tsx           | 100        | 100      | 100       | 100    ||
| **src/pages/[redirect]**     | 85.29      | 75       | 66.66     | 90.32  ||
| index.tsx                    | 85.29      | 75       | 66.66     | 90.32  | 30-31,37 |
| **src/services**             | 100        | 100      | 100       | 100    ||
| api.tsx                      | 100        | 100      | 100       | 100    ||
 
### Screenshots

https://github.com/Real-Dev-Squad/tiny-site-frontend/assets/70854507/75fb3c5c-f5b9-4448-99e3-9ca5d789ebe6

## Additional Notes
N/A

### CODE REVIEW

File: src/services/api.tsx [hunk 1]
```diff
@@ -16,4 +20,17 @@ const useAuthenticatedQuery = () => {
     });
 };
 
-export { useAuthenticatedQuery };
+
```
Comment (ajoykumardas12) [inline src/services/api.tsx:23]: Is extra empty line intended here?

File: src/services/api.tsx [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import axios from 'axios';
 import { useQuery } from 'react-query';
 
-import { TINY_API_URL } from '@/constants/url';
+
+import { TINY_API_URL, TINY_API_URL_DETAIL } from '@/constants/url';
+
+
+
```
Comment (ajoykumardas12) [inline src/services/api.tsx:8]: Is extra empty line intended here?

File: src/services/api.tsx (hunk 1 above)
Comment (sahsisunny) [reply src/services/api.tsx]: Removed

**Comment by ajoykumardas12** [comment]
Tested in staging should be no I think.
Do we need to add test for the scenario 'premium user and clicks on Go button'? Rest looks good to me.

**Comment by sahsisunny** [comment]
> Tested in staging should be no I think. Do we need to add test for the scenario 'premium user and clicks on Go button'? Rest looks good to me.

https://github.com/Real-Dev-Squad/tiny-site-frontend/pull/73/files#diff-e2a07f6d658313f1dc102c684c9b1721e64df1016d19fc74f1fe862848e395b0R93

**Review by ajoykumardas12 — APPROVED** [review]

**Comment by sahsisunny** [comment]
> > https://github.com/Real-Dev-Squad/tiny-site-frontend/pull/73/files#diff-e2a07f6d658313f1dc102c684c9b1721e64df1016d19fc74f1fe862848e395b0R93
> 
> This tests for non-premium user, but works I guess since there are only two conditions.

IsPremiumUser is by default false and its hard coded we can't test for true condition

**Review by vinit717 — APPROVED** [review]

**Pull request merged**